frontend/src/modules/Metadata_Forms/components/createMetadataFormModal.js

```diff
@@ -183,8 +154,32 @@ export const CreateMetadataFormModal = (props) => {
           initialValues={{
             name: '',
             description: '',
-            visibility: visibilityDict.Global
+            visibility: visibilityDict.Global,
+            owner: '',
+            environment: '',
+            group: '',
+            organization: ''
           }}
+          validationSchema={Yup.object().shape({
+            name: Yup.string()
+              .max(255)
+              .required('*Metadata forms name is required'),
+            description: Yup.string().max(200),
+            owner: Yup.string().required('*Owner is required'),
```

Comment:
optionally can enforce owner to be one of user's groups using `owner: Yup.string().oneOf(groups).required('*Owner is required'),`

otherwise a user could theoretically free form type in a random group name as of now that does not exist or a user is not a part of

Reply:
For I don't want to make you review this PR one more time, I will place this additional validation in next PR)